mappings/net/minecraft/world/ChunkRegion.mapping

```diff
@@ -13,6 +13,7 @@ CLASS net/minecraft/class_3233 net/minecraft/world/ChunkRegion
 	FIELD field_14097 levelProperties Lnet/minecraft/class_31;
 	FIELD field_14098 chunks Ljava/util/List;
 	FIELD field_14099 blockTickScheduler Lnet/minecraft/class_1951;
+	FIELD field_20668 biomeAccessor Lnet/minecraft/class_4543;
```

Comment:
`biomeAccess`? Or is this something else?

Reply:
Nice spotting, fixing.